features/environment.py

```diff
@@ -9,16 +13,18 @@ def before_all(context):
                                                 tag=context.container_tag)
 
 
-def after_all(context):
+def after_scenario(context, step):
     try:
         context.client.containers.get(context.container_name).kill()
     except docker.errors.NotFound:
         pass
 
 
-def after_step(context, step):
-    if step.status == "failed":
-        try:
-            context.client.containers.get(context.container_name).kill()
-        except docker.errors.NotFound:
-            pass
+fixture_registry = {
+    "fixture.browser.chrome":  browser_chrome,
```

Comment:
is Chrome the browser on the Pi?

Reply:
because if not, would be better to use the browser on the pi, but could be a ticket for next sprint

Reply:
Pi has Chromium installed by default